grails-app/controllers/au/org/emii/portal/WpsController.groovy

```diff
@@ -63,7 +64,7 @@ class WpsController extends HostVerifyingController {
                 job: [
                     uuid: params.uuid,
                     reportUrl: g.createLink(action: 'jobReport', absolute: true, params: params),
-                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)),
+                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)).withZone( DateTimeZone.forID( "Australia/Melbourne" )),
```

Comment:
The server shouldn't be hard-coding the timezone for the client. In the past we always invested the effort to fix time/zone formatting issues - and without resorting to coding the behavior on the server. So there is already a precedent in terms of effort that's been invested in avoiding this kind of change. @jonescc any thoughts?